### ISSUE TITLE
Workspace.getEditors is deprecated.

### ISSUE THREAD

**Issue by halkun** [mpeterson2/save-session#112]
Use Workspace::getTextEditors instead
```
Workspace.getEditors (C:\Users\User\AppData\Local\atom\app-0.199.0\resources\app.asar\src\workspace.js:977:7)
Object.restoreText (C:\Users\User\.atom\packages\save-session\lib\files.coffee:53:31)
Object.restore (C:\Users\User\.atom\packages\save-session\lib\files.coffee:41:7)
<unknown> (C:\Users\User\.atom\packages\save-session\lib\files.coffee:16:13)
FSReqWrap.oncomplete (fs.js:379:3)
```
Use Workspace::getTextEditors instead
```
Workspace.getEditors (C:\Users\User\AppData\Local\atom\app-0.199.0\resources\app.asar\src\workspace.js:977:7)
Object.restoreText (C:\Users\User\.atom\packages\save-session\lib\files.coffee:46:31)
Object.restore (C:\Users\User\.atom\packages\save-session\lib\files.coffee:41:7)
<unknown> (C:\Users\User\.atom\packages\save-session\lib\files.coffee:16:13)
FSReqWrap.oncomplete (fs.js:379:3)
```

**Comment by mpeterson2** [comment]
Fixed in `0.15.1`. Thanks.

**Comment by halkun** [comment]
Thanks for the quick response and update!
